mobile/lib/components/ra_bottomnavbar.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:radioaktywne/extensions/build_context.dart';
+
+import 'package:radioaktywne/resources/assets.gen.dart';
+import 'package:radioaktywne/resources/colors.dart';
+
+class RaBottomNavigationBar extends StatelessWidget {
+  const RaBottomNavigationBar({
+    super.key,
+    required this.currentIndex,
+  });
+
+  final int currentIndex;
+
+  static final labels = [
+    'home',
+    'mic',
+    'album',
+    'article',
+  ];
+
+  SizedBox _loadIconWithName(String name) {
+    return SizedBox(
+      width: 35,
+      height: 35,
+      child: SvgGenImage('assets/icons/$name.svg').svg(),
+    );
+  }
+
+  List<BottomNavigationBarItem> _makeList(List<String> labels) {
+    final list = <BottomNavigationBarItem>[];
+    for (final label in labels) {
+      list.add(
+        BottomNavigationBarItem(
+          icon: _loadIconWithName('${label}_outline'),
+          activeIcon: _loadIconWithName(label),
+          label: label,
+        ),
+      );
+    }
+    return list;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.only(top: 8),
+      height: 79,
+      color: RAColors.of(context).highlightGreen,
+      child: BottomNavigationBar(
+        showSelectedLabels: false,
+        showUnselectedLabels: false,
+        type: BottomNavigationBarType.fixed,
+        backgroundColor: context.colors.backgroundDark,
+        selectedItemColor: context.colors.highlightGreen,
+        unselectedItemColor: context.colors.highlightGreen,
+        currentIndex: currentIndex,
+        onTap: (index) {
+          // TODO: Implement navigation
+        },
+        items: _makeList(labels),
```

Comment:
items should look somewhat like this (hard-coded or as a function output):
```suggestion
        items: const [
          BottomNavigationBarItem(
            icon: Icon(Icons.home_outlined),
            activeIcon: Icon(Icons.home),
            label: 'home',
          ),
          BottomNavigationBarItem(
            icon: Icon(Icons.mic_none_outlined),
            activeIcon: Icon(Icons.mic),
            label: 'mic',
          ),
          BottomNavigationBarItem(
            icon: Icon(Icons.album_outlined),
            activeIcon: Icon(Icons.album),
            label: 'album',
          ),
          BottomNavigationBarItem(
            icon: Icon(Icons.article_outlined),
            activeIcon: Icon(Icons.article),
            label: 'article',
          ),
        ],
```
Implementing it with a mapping function like you did here probably won't be so straight forward with this approach unfortunately, since it's hard to get `Icons.<insert appropriate icon name here>` based on icon names as strings. Also, notice the `mic_none_outlined` in the mic section. For some reason `mic_outlined` is basically the same as `mic`. Why you ask? Good question, I'm wondering too. I think we can leave the whole list hard-coded here since there's gonna be one always-the-same-looking bottomnavbar for the whole app. @korkuck @tymwitko what are your opinions?